federation/cmd/federation-apiserver/app/import_known_versions.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package app
+
+// These imports are the API groups the client will support.
+import (
+	"k8s.io/kubernetes/federation/apis/core"
+	coreinstall "k8s.io/kubernetes/federation/apis/core/install"
+	federationinstall "k8s.io/kubernetes/federation/apis/federation/install"
+	"k8s.io/kubernetes/pkg/api"
+	autoscalinginstall "k8s.io/kubernetes/pkg/apis/autoscaling/install"
+	batchinstall "k8s.io/kubernetes/pkg/apis/batch/install"
+	extensionsinstall "k8s.io/kubernetes/pkg/apis/extensions/install"
+)
+
+func init() {
```

Comment:
Can we move these install calls where the scheme+registery are actually declared?